libcnb-test/src/build.rs

```diff
@@ -3,31 +3,20 @@ use libcnb_cargo::cross_compile::{cross_compile_assistance, CrossCompileAssistan
 use libcnb_cargo::{
     assemble_buildpack_directory, build_buildpack_binary, BuildError, CargoProfile,
 };
-use std::path::{Path, PathBuf};
+use std::path::PathBuf;
 use tempfile::{tempdir, TempDir};
 
 /// Packages the current crate as a buildpack into a temporary directory.
 pub(crate) fn package_crate_buildpack(
     target_triple: impl AsRef<str>,
-    app_dir: impl AsRef<Path>,
 ) -> Result<TempDir, PackageCrateBuildpackError> {
     let cargo_manifest_dir = std::env::var("CARGO_MANIFEST_DIR")
         .map(PathBuf::from)
         .map_err(PackageCrateBuildpackError::CannotDetermineCrateDirectory)?;
 
-    let temp_app_dir =
-        tempdir().map_err(PackageCrateBuildpackError::CannotCreateAppTempDirectory)?;
-
     let mut copy_options = CopyOptions::new();
     copy_options.content_only = true;
```

Comment:
I think `copy_options` is now also unused?

Reply:
By the power of force-pushing, I don't know what you're talking about. 😬